src/lib/factories/listing.ts

```diff
@@ -266,6 +269,9 @@ export function EmptyListing(): ListingType {
     fullTankIncluded: undefined,
     expertInstructionIncluded: undefined,
     otherServices: undefined,
+    useDefaultAdditionalServices: undefined,
+    useDefaultDescription: undefined,
+    useDefaultWarranty: undefined,
```

Comment:
wouldn't that contradict the type definition below?

Reply:
sorry, what?

Reply:
I might be misinterpreting things here, but the types say:
```
  useDefaultAdditionalServices: boolean
  useDefaultDescription: boolean
  useDefaultWarranty: boolean
```

and therefore the value can't be null/undefined. no?

Reply:
check this file https://github.com/carforyou/carforyou-api-client-pkg/blob/master/src/lib/factories/listing.ts from MASTER. is about emptyListing.

Reply:
I don't know what that , but as you're confident, let's roll!

Reply:
@masone in typescript `undefined` and `null` are so-called bottom type. That means they are a bit special - they belong to every type that exist.

You can write:
```typescript
const foo: boolean = undefined
```
and you won't get a type error.

The concept behind this is so that you can handle not-initialized variables or empty values with ease, which is exactly what we're doing here.

Reply:
Sidenote: thats what we plan to disable in the future https://autoricardo.atlassian.net/browse/CAR-5843 (